dev/ci/ci-hott.sh

```diff
@@ -7,4 +7,4 @@ HoTT_CI_DIR=${CI_BUILD_DIR}/HoTT
 
 git_checkout ${HoTT_CI_BRANCH} ${HoTT_CI_GITURL} ${HoTT_CI_DIR}
 
-( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && make -j ${NJOBS} )
+( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && ./etc/coq-scripts/timing/make-pretty-timed-or-error.sh -j ${NJOBS} && make validate )
```

Comment:
Ah, well, if this target works before we come to a decision on #733, I can strip off the timing commit from this PR, and we can merge the validate commit without it.

Is the issue that you're worried that the build might fail because of a bad script and it'll be hard to diagnose?  What do you think of this alternative?
```bash
( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && (make -j ${NJOBS} TIMED=1 | tee time-of-build.log; exit ${PIPESTATUS[0]}) && make validate )
python "./etc/coq-scripts/timing/make-one-time-file.py" time-of-build.log time-of-build-pretty.log && cat time-of-build-pretty.log
```
This way we separate the running of the test (which is done entirely by `make`) from the timing display (done by the submodule).